Cargo.toml

```diff
@@ -547,6 +547,7 @@ path = "src/bin/coreutils.rs"
 name = "uudoc"
 path = "src/bin/uudoc.rs"
 required-features = ["uudoc"]
+# cargo run --bin uudoc --features uudoc
```

Comment:
This is an unusual place to document a command. Is there a particular reason?

Reply:
yes please :)

Reply:
done